vulnerabilities/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This should be deleted and then added to the `.gitignore`

Reply:
Oops. Done